doc/source/scalability/innodb_io.rst

```diff
@@ -4,39 +4,19 @@
  Improved |InnoDB| I/O Scalability
 ===================================
 
-Because |InnoDB| is a complex storage engine it must be configured properly in order to perform at its best. Some points are not configurable in standard |InnoDB|. The goal of this feature is to provide a more exhaustive set of options for |XtraDB|.
+Because |InnoDB| is a complex storage engine it must be configured properly in
+order to perform at its best. Some points are not configurable in standard
+|InnoDB|. The goal of this feature is to provide a more exhaustive set of
+options for |XtraDB|.
 
 Version Specific Information
 ============================
 
-  * :rn:`5.7.10-1`
-
-    * Feature ported from |Percona Server| 5.6
+  * :rn:`8.0.12-1` - the feature was ported from |Percona Server| 5.7.
    
 System Variables
 ================
 
-
-.. variable:: innodb_use_global_flush_log_at_trx_commit
-
-   :cli: Yes
-   :conf: Yes
-   :scope: Global
-   :dyn: Yes
-   :type: Boolean
-   :default: True
-   :range: True/False
-
-This variable is used to control the ability of the user to set the value of the global |MySQL| variable :variable:`innodb_flush_log_at_trx_commit`.
-
-If :variable:`innodb_use_global_flush_log_at_trx_commit` is set to ``0`` (False), the client can set the global |MySQL| variable, using: 
-
-.. code-block:: mysql
-
-  SET innodb_use_global_flush_log_at_trx_commit=N
-
-If :variable:`innodb_use_global_flush_log_at_trx_commit` is set to ``1`` (True), the user session will use the current value of :variable:`innodb_flush_log_at_trx_commit`, and the user cannot reset the value of the global variable using a ``SET`` command.
-
 .. variable:: innodb_flush_method
```

Comment:
With ALL_O_DIRECT removed, are there any Percona differences from upstream in innodb_flush_method? If no, remove the section

Reply:
According to our documentation, parallel doublewrite files are affected. In the upstream documentation these files are not mentioned.

Reply:
OK to discuss parallel doublewrite flushing only in its feature docs (left same comment on @hrvojem PR)

Reply:
Comment not addressed

Reply:
Move the variable innodb_flush_method to the section "Parallel doublewrite buffer" in XtraDB Performance Improvements for I/O-Bound Highly-Concurrent Workloads